plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryThumbnails.php

```diff
@@ -119,7 +121,11 @@ protected function should_display_view_all( $mode, $thumbnails_count, $product_g
 	 * @return string Rendered block type output.
 	 */
 	protected function render( $attributes, $content, $block ) {
-		if ( isset( $block->context['thumbnailsPosition'] ) && '' !== $block->context['thumbnailsPosition'] && 'off' !== $block->context['thumbnailsPosition'] ) {
+		if ( ! isset( $block->context ) ) {
+			return '';
+		}
+
+		if ( isset( $block->context['thumbnailsPosition'] ) && '' !== $block->context['thumbnailsPosition'] ) {
```

Comment:
This function still silently fails if context exists but `$block->context['thumbnailsPosition']` does not. We should do one of two things here:
1. Reverse this condition and early-return an empty string
2. Add an `else` which returns an empty string

Option 1 would be preferable.

Reply:
Good catch!

https://github.com/woocommerce/woocommerce/pull/54682/commits/f0bdf28af9ef1ece32b149026450928307d8de29